packages/nouns-contracts/test/foundry/rewards/ProposalRewards.t.sol

```diff
@@ -709,6 +771,51 @@ contract VotesRewardsTest is BaseProposalRewardsTest {
         assertEq(rewards.getVotingClientIds(proposalId), expectedClientIds);
     }
 
+    function test_getVotingClientIds_filtersProposalsBelowEligibilityQuorum() public {
+        // clientId1 should not be included because proposal only had against votes
+        vote(bidder1, proposalId, 0, 'against', clientId1);
+        mineBlocks(VOTING_PERIOD);
+
+        uint32 proposalId2 = uint32(propose(bidder1, address(1), 1 ether, '', '', 'my proposal', 0));
+        mineBlocks(VOTING_DELAY + UPDATABLE_PERIOD_BLOCKS + 1);
+        vote(bidder1, proposalId2, 1, 'i support', clientId2);
+        mineBlocks(VOTING_PERIOD);
+        uint32[] memory votingClientIds = rewards.getVotingClientIds(proposalId2);
+
+        // doesn't revert
```

Comment:
why would we expect it to revert?

e.g. if I'm thinking about the check that every client has non-zero votes, both clients 1 and 2 have votes here, so they would not fail that check.

can you specify what check would have reverted but does not thanks to the filtering working?

Reply:
added a comment to explain: 7b9e962b39815f8a6b96fc609c151749b2f2f5cb